patterns/1-initial/scoring-grid.md

```diff
@@ -0,0 +1,114 @@
+## Title
+
+Software Project Scoring Grid
+
+## Patlet
```

Comment:
The Patlet is meant to allow readers to quickly scan a lot of Problem-Solution pairs, to find the things relevant for their orgs.

If we were to rewrite this Patlet as two fairly short sentences, how would that look like?
1st sentence Problem
2nd sentence Solution